reconftw.cfg

```diff
@@ -112,6 +112,10 @@ SPRAY=true
 COMM_INJ=true
 PROTO_POLLUTION=true
 
+# Source code review
+LGTM_CLI=true
+LGTM_CLI_REPO="gagliardetto/lgtm-cli"
```

Comment:
Let the user change the repo in case it changes anything. Perhaps we could also specify a branch and checkout to it here: https://github.com/six2dez/reconftw/blob/c1b3e86204626fe27e186704b809aaf5c98dd57f/install.sh#L258